mlflow/tracking/_tracking_service/client.py

```diff
@@ -376,7 +434,7 @@ def log_batch(self, run_id, metrics=(), params=(), tags=()):
 
         param_batches = chunk_list(params, MAX_PARAMS_TAGS_PER_BATCH)
         tag_batches = chunk_list(tags, MAX_PARAMS_TAGS_PER_BATCH)
-
+        run_operations_list = []  # type: List[RunOperations]
```

Comment:
Let's add comment explain its purpose, which is not obvious from the variable name.